app/src/main/java/com/infomaniak/drive/ui/fileList/fileDetails/FileDetailsCommentsFragment.kt

```diff
@@ -80,7 +80,7 @@ class FileDetailsCommentsFragment : FileDetailsSubFragment() {
                     fileDetailsViewModel.postFileComment(currentFile, name).observe(viewLifecycleOwner) { apiResponse ->
                         if (apiResponse.isSuccess()) {
                             apiResponse?.data?.let { comment ->
-                                commentsAdapter.addComment(comment)
```

Comment:
Now the function "addComment" is never used !
Why did you do it like that?

Reply:
I tried replacing `addComment()` with `addAll()` because the later one contains `hideLoading()` so placeholders wouldn't stay displaying if a use inputs a comment before they finish loading. But this actually doesn't make much sense now that you're asking me about it so I just reverted it.